DSCResources/MSFT_xCertReq/MSFT_xCertReq.psm1

```diff
@@ -128,7 +128,7 @@ function Get-TargetResource
     )
 
     # The certificate authority, accessible on the local area network
-    $ca = "'$CAServerFQDN\$CARootName'"
+    $ca = "$CAServerFQDN\$CARootName"
```

Comment:
Just all changed lines as discussed under PR 43 and issue 42. Also documented the changes in README.md file as requested.